### DISCUSSION TITLE
is there a possibility that only certain file types can be sent with a report?

### DISCUSSION THREAD

**Discussion by JimpoTEDY** [globaleaks/globaleaks-whistleblowing-software#3563] [Q&A]
### Proposal

Hello,

is there a possibility that only certain file types can be sent with a report?

### Motivation and context

Certain file types may contain malicious code

**Comment by evilaliv3** [comment]
Thank you.

This is a recurrent question. 
Within the GlobaLeaks community we currently consider that any file can be an evidence and even wanting to prevent to upload malicious file (e.g. exe), users uploading a zip file will be able to bypass the filter easily.

This condition can be solved with Antiviruses and best practices of operation including eventually opening evidences files on an air-gapped machines

**Comment by maelaborazioni** [comment]
Some companies do not allow photos to be taken inside the company so you may want to prevent images from being uploaded...
